pkg/agent/nodeportlocal/k8s/npl_controller.go

```diff
@@ -137,10 +137,29 @@ func (c *NPLController) Run(stopCh <-chan struct{}) {
 	for i := 0; i < numWorkers; i++ {
 		go wait.Until(c.Worker, time.Second, stopCh)
 	}
-
+	go c.SyncRules(stopCh)
```

Comment:
This runs `c.SyncRules` in a goroutine, but `SyncRules` itself starts a new goroutine, so it doesn't seem right...

Reply:
Fixed